etc/sca/windows/Untitled4

```diff
@@ -0,0 +1,641 @@
+# Security Configuration assessment
+# CIS Checks for Windows 2012 R2 Member Server L2
+# Copyright (C) 2015-2019, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation
+#
+# Based on Center for Internet Security Benchmark for Microsoft Windows Server 2012 R2 v2.3.0
+
+policy:
+  id: "cis_win_server_memberL2"
+  file: "cis_win_server_memberL2_rcl.yml"
+  name: "CIS benchmark for Windows Member Server L2"
+  description: "This document provides prescriptive guidance for establishing a secure configuration posture for Microsoft Windows Server."
+  references:
+    - https://www.cisecurity.org/cis-benchmarks/
+
+requirements:
+  title: "Check that the Windows platform is Windows Server "
+  description: "Requirements for running the CIS benchmark Member Server L2 under Windows Server "
+  condition: "any required"
+  rules:
+     - 'r:HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion -> ProductName -> r:^Windows Server 2012 R2 ;'
+     - 'r:HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion -> ProductName -> r:^Windows Server 2016 Datacenter;'
+     - 'r:HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion -> ProductName -> r:^Windows Server 2019 Datacenter;'
+
+checks:
+# Section 2.3.7 - Interactive logon
+  - id: 9500
```

Comment:
ID 9500 and subsequent are already in use https://github.com/wazuh/wazuh/blob/67ecaa64da8c6b92b2aad16f48b0ba0cc11ab007/etc/sca/applications/cis_mysql5-6_community.yml#L30. Following the SCA documentation, they have to be unique.